### ISSUE TITLE
[Bug Report] A clean new Umami APP can't login ver1.39.1 

### ISSUE THREAD

**Issue by eallion** [umami-software/umami#1586]
```
umami-umami-1  | ✓ Database is up to date.
umami-umami-1  | $ node scripts/update-tracker.js
umami-umami-1  | $ node server.js
umami-umami-1  | Listening on port 3000
umami-db-1     | 2022-10-25 06:06:02.796 UTC [32] ERROR:  column account.account_uuid does not exist at character 197
umami-db-1     | 2022-10-25 06:06:02.796 UTC [32] STATEMENT:  SELECT "public"."account"."user_id", "public"."account"."username", "public"."account"."password", "public"."account"."is_admin", "public"."account"."created_at", "public"."account"."updated_at", "public"."account"."account_uuid" FROM "public"."account" WHERE ("public"."account"."username" = $1 AND 1=1) LIMIT $2 OFFSET $3
umami-umami-1  | PrismaClientKnownRequestError: 
umami-umami-1  | Invalid `prisma.account.findUnique()` invocation:
umami-umami-1  | 
umami-umami-1  | 
umami-umami-1  | The column `account.account_uuid` does not exist in the current database.
umami-umami-1  |     at RequestHandler.handleRequestError (/app/node_modules/@prisma/client/runtime/index.js:30873:13)
umami-umami-1  |     at RequestHandler.request (/app/node_modules/@prisma/client/runtime/index.js:30856:12)
umami-umami-1  |     at async PrismaClient._request (/app/node_modules/@prisma/client/runtime/index.js:31836:16)
umami-umami-1  |     at async __WEBPACK_DEFAULT_EXPORT__ (/app/.next/server/pages/api/auth/login.js:79:21)
umami-umami-1  |     at async Object.apiResolver (/app/node_modules/next/dist/server/api-utils/node.js:366:9)
umami-umami-1  |     at async NextNodeServer.runApi (/app/node_modules/next/dist/server/next-server.js:481:9)
umami-umami-1  |     at async Object.fn (/app/node_modules/next/dist/server/next-server.js:735:37)
umami-umami-1  |     at async Router.execute (/app/node_modules/next/dist/server/router.js:247:36)
umami-umami-1  |     at async NextNodeServer.run (/app/node_modules/next/dist/server/base-server.js:347:29)
umami-umami-1  |     at async NextNodeServer.handleRequest (/app/node_modules/next/dist/server/base-server.js:285:20) {
umami-umami-1  |   code: 'P2022',
umami-umami-1  |   clientVersion: '4.5.0',
umami-umami-1  |   meta: { column: 'account.account_uuid' }
umami-umami-1  | }
```

**Comment by mskian** [comment]
check my solution - https://github.com/umami-software/umami/issues/1583#issuecomment-1290037343

**Comment by mskian** [comment]
check-db and run migrate it

**Comment by mskian** [comment]
try this

```sh
docker exec -it umami-umami-1 yarn prisma migrate resolve --applied 04_add_uuid
```

**Comment by mskian** [comment]
did u restart the database after migration?

**Comment by briancao** [comment]
Fixed in 1.39.2.  Will be releasing tonight.

**Comment by mikecao** [comment]
Fixed in v1.39.2.
